### PR TITLE
parse error

### PR DESCRIPTION
**Pull request by LarsTepper**

### CODE REVIEW

File: simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs [hunk 1]
```diff
@@ -16,6 +17,10 @@ public override void SetValue(IDbDataParameter parameter, PreciseDecimal value)
 
         public override PreciseDecimal Parse(object value)
         {
+            if(value.GetType() == typeof(decimal))
```
Comment (simplicbe) [inline simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs]: Hmm, `value is decimal` is much faster...

Comment (yvz5) [reply simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs]: https://stackoverflow.com/questions/686412/c-sharp-is-operator-performance

Comment (simplicbe) [reply simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs]: @yvz5 the problem is, we have unboxing here and no cast...

Comment (LarsTepper) [reply simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs]: ok, i'm old school ...

**Pull request merged**